jbpm-installer/src/test/resources/ddl60/db2/db2-jbpm-schema.sql

```diff
@@ -130,6 +130,7 @@
         processInstanceId bigint not null,
         type integer not null,
         workItemId bigint,
+        cancel smallint not null,
```

Comment:
do we need to update ddl60 schemas? my understanding is that this is used to test migration from 6.0 to 7.x

Reply:
this flag will be changed in exchange for new type of nodeinstancelog